File: packages/insomnia-httpsnippet/test/fixtures/output/go/native/text-plain.go
package main

import (
	"fmt"
	"strings"
	"net/http"
	"io/ioutil"
)

func main() {

	url := "http://mockbin.com/har"

	payload := strings.NewReader("Hello World")

	req, _ := http.NewRequest("POST", url, payload)

	req.Header.Add("content-type", "text/plain")

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}

File: packages/insomnia-httpsnippet/test/fixtures/output/node/unirest/text-plain.js
var unirest = require('unirest');

var req = unirest('POST', 'http://mockbin.com/har');

req.headers({
  'content-type': 'text/plain'
});

req.send('Hello World');

req.end(function(res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});
